app/controllers/developers_controller.rb

```diff
@@ -0,0 +1,66 @@
+class DevelopersController < ApplicationController
+  skip_before_action :ensure_current_user
+
+
+  def index
+    @developers = Developer.all
+    # NEXT STEP - order by rating
+  end
+
+  def show
+    @developer = Developer.find_by(id: params[:id])
```

Comment:
You might prefer to use `find(params[:id])` - so that rails throws a 404 if the record doesn't exist.